### PR TITLE
Ref: Refactored Connect Wallet Modal with new theme colours and text

### LINKED ISSUE #2193: Use new text styles and color variables for Modal Connect Wallet (0.5 Day)
Now that we have clearly defined text & color style guide we should adopt the Connect Wallet Modal with the new styles

[Figma Spec](https://www.figma.com/file/R056tSYMjvfwaFidEBItYy/Widgets-V2?node-id=3832%3A241)

### PR DESCRIPTION
**Pull request by Frankaus**
closes #2193

### CODE REVIEW

**Review by pimato — CHANGES_REQUESTED** [review]
In the `request permission on metamask` view we have a headline with a wrong black color, should be `text1`:
Bildschirmfoto 2021-08-17 um 14 39 33

File: app/src/components/modal/modal_connect_wallet/index.tsx [hunk 1]
```diff
@@ -129,7 +103,9 @@ const ConnectButton = (props: ButtonProps) => {
     <ButtonStyled buttonType={ButtonType.secondaryLine} disabled={disabled} onClick={onClick}>
       <ButtonLeft>
         {icon}
-        <Text>{text}</Text>
+        <TYPE.bodyRegular color={'text1'} lineHeight={'1.2'} margin={'0px'}>
```
Comment (pimato) [inline app/src/components/modal/modal_connect_wallet/index.tsx:106]: please remove the lineHeight as bodyRegular already has it

File: app/src/components/modal/modal_connect_wallet/index.tsx [hunk 2]
```diff
@@ -225,11 +201,23 @@ export const ModalConnectWallet = (props: Props) => {
...
+          <TYPE.heading3 color={'text1'} marginBottom={'48px'} marginTop={'16px'}>
+            {isConnectingToWallet ? 'Unlock Wallet' : 'Connect a Wallet'}
+          </TYPE.heading3>
           {isConnectingToWallet ? (
             <>
               <Spinner big={true} />
-              <ConnectingText>{connectingText}</ConnectingText>
+              <TYPE.bodyRegular
+                color={'text2'}
+                fontWeight={'normal'}
```
Comment (pimato) [inline app/src/components/modal/modal_connect_wallet/index.tsx:212]: remove

File: app/src/components/modal/modal_connect_wallet/index.tsx [hunk 3]
```diff
@@ -225,11 +201,23 @@ export const ModalConnectWallet = (props: Props) => {
...
+                letterSpacing={'0.4px'}
```
Comment (pimato) [inline app/src/components/modal/modal_connect_wallet/index.tsx:213]: remove

File: app/src/components/modal/modal_connect_wallet/index.tsx [hunk 4]
```diff
@@ -225,11 +201,23 @@ export const ModalConnectWallet = (props: Props) => {
...
+                lineHeight={'1.5'}
```
Comment (pimato) [inline app/src/components/modal/modal_connect_wallet/index.tsx:214]: bodyRegular has already a lineHeight

**Review by pimato — CHANGES_REQUESTED** [review]

**Review by pimato — APPROVED** [review]

**Review by Mi-Lan — APPROVED** [review]
LGTM!

**Review by pimato — APPROVED** [review]

**Pull request merged**